docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc

```diff
@@ -18,32 +15,48 @@ For more information on Red Hat Technology Preview features support scope,
+* Predictive Model Markup Language (PMML) - The PMML feature is fully implementated for the following predictive model types:
+** Decision Tree
+** Regression
+** Scorecard
 +
-The *Read* button turns blue.
-. Click *Save*. The DevOps -> *Deployments* link is now available on the Home page.
+PMML has a partial implementation of the Mining Model type; with the following model sub-types available:
+** Select All
+** Select First
+** Model Chaining
+
 +
+*Note:* The following Technology Preview features are experimental and may require additional API changes:
+* Executable Model
+* Multithreaded Execution - Solves a single data set by partitioning across multiple threads. It is turned off by default and can be enabled either with a KieBaseConfiguration option or with a system property as explained here http://docs.jboss.org/drools/release/7.5.0.Final/drools-docs/html_single/#_multithreaded_rule_engine
+* Rule Units - Rule Units represent a declarative approach to partitioning a rule set in to smaller units, binding different data sources to those units, and the execution of the individual unit. A Rule Unit is comprised of data sources, global variables, and rules.
+
+//There isn't any specific option to be enabled to start using this feature. It's sufficient to declare the unit to which the rules belong at the beginning of the drl file and trigger the execution through a RuleUnitExecutor instead of using a KieSession. You can find more detail about this here http://docs.jboss.org/drools/release/7.5.0.Final/drools-docs/html_single/#_rule_units_2
+
+//Mario Fusco - Hi Michele, I agree with Lukas' comment: Rule units, Multithreaded engine and Executable model are all experimental features even if with different levels of maturity. In particular while the first 2 are ready to be used (even if we don't provide any support for them yet) the Executable model is still a work in progress. It is not completed and not really usable at the moment. I don't know if it makes sense to underline this difference (or mentioning the Executable model at all at this stage). I'm also CC'ing Mark Proctor in case he has something to add to this. Regards, Mario
+//Mark Proctor-to me, Mario, Lukas
+//Yes leave them in them in the experimental list, but do emphasise that while these have a degree of stability in them (shouldn't crash), we need more time to have potential API changes.
+//Mark Proctor - Yes, they are experimental and subject to api changes. The are maturing beta quality features.
```

Comment:
Can you please edit lines 43-48 to remove people's names?

Reply:
Done. Thank you Edson :)

Reply:
Done.